doc/manual/overview.rst

```diff
@@ -128,7 +128,7 @@ and Github users mpsijm, sylxjtu.
 Some code has been ported from the ETH Zurich fork by Christoph
 Krautz, Thomas Rast et al.
 
-DOMjudge is Copyright (c) 2004 - 2020 by the DOMjudge developers and contributors.
+DOMjudge is Copyright (c) 2004 - |today| by the DOMjudge developers and contributors.
```

Comment:
Just to confirm: `|today|` only prints a year, not a full date?

Reply:
Yes: https://github.com/DOMjudge/domjudge/pull/710/files#diff-3823844ce2f5e2ce64bf66dd74002ad2R64